server/controllers/index.js

```diff
@@ -1,7 +1,9 @@
 const express = require("express")
 const router = express.Router()
-const { getPlaces } = require('./places')
+const { getPlaces } = require("./places")
+const { getGuides } = require("./guides")
 
-router.get('/places', getPlaces)
+router.get("/places", getPlaces)
```

Comment:
this is a cleaner way of writing your router.
```suggestion
router.route("/places")
    .get(getPlaces)
```

Reply:
you don't have to do this one.